version.c

```diff
@@ -41,9 +53,6 @@ const char ruby_revision[] = RUBY_FULL_REVISION;
 const char ruby_release_date[] = RUBY_RELEASE_DATE;
 const char ruby_platform[] = RUBY_PLATFORM;
 const int ruby_patchlevel = RUBY_PATCHLEVEL;
-const char ruby_description[] = RUBY_DESCRIPTION_WITH("");
```

Comment:
You will want to keep this for compatibility reasons since it's exposed in public headers:
https://github.com/ruby/ruby/blob/35e111fd3e38ee6f44cd67b255815ef301ac5211/include/ruby/version.h#L142

In general only headers that are under `include/` are exposed, so `/version.h` is not public.

Reply:
We can keep it - but note it's broken - it doesn't include JIT status.

Reply:
It is a static information, but JIT status is a dynamic information.